apps/maestro/src/config/chains/evm-chains.ts

```diff
@@ -166,6 +165,26 @@ const hyperEVMTestnet = defineChain({
   testnet: true,
 });
 
+const celoSepolia = defineChain({
```

Comment:
I can see that this is also defined in `packages/evm/src/clients/custom/celo.ts`, do we need both?